x-pack/test/functional_with_es_ssl/apps/cases/common/roles.ts

```diff
@@ -59,6 +59,30 @@ export const casesNoDelete: Role = {
   },
 };
 
+export const casesReadSettings: Role = {
```

Comment:
Using the same naming convention as the other roles it's hard to come up with something good, but maybe `casesReadAndEditSettings`?

Reply:
> I am bad at naming variables 😄

Nah it made sense with the existing ones like `CasesReadDelete` for example. It is just misleading because not all the verbs are subfeature privileges here 🤷 

It's just a nit anyway so no need to change. `casesReadAndEditSettings` sounds good

Reply:
[Updated](https://github.com/elastic/kibana/pull/202053/commits/17f36827862f57a35119fa82ce404b75ada16c93#diff-1a98417c4b888d627729a7c2916fb9d774ef7c2f6e195b771950e5bd77b0e0d4)